.travis.yml

```diff
@@ -2,14 +2,16 @@ language: node_js
 sudo: required
 node_js:
   - "0.10"
+  - "4"
   - "4.4.3"
+  - "6"
+  - "8"
 services:
   - docker
 before_install:
   - sudo apt-get update
   - sudo apt-get install --assume-yes apache2-utils
   - npm install -g npm@2.13.5
-  - npm install -g grunt-cli
```

Comment:
Removed this since it's now in "devDependencies"

Reply:
👎 it is for the CI process to run the script and test the project to allow the PR is required the grunt-cli

Reply:
👎 it is for the CI process to run the script and test the project to allow the PR is required the grunt-cli